packages/stylelint-config-ckeditor5/.stylelintrc

```
@@ -1,10 +1,16 @@
 {
 	"extends": "stylelint-config-recommended",
+	"plugins": [
+		"ckeditor5-plugin"
```

```suggestion
		"stylelint-plugin-ckeditor5-rules/lib/rules/license-header"
```

This version works for me.